src/Components/CardSection.js

```diff
@@ -1,35 +1,31 @@
 import React, {Component} from 'react'
 import MovieCard from './MovieCard'
-class CardSection extends Component {
-  constructor() {
-    super();
-    this.state = {
-      movies: null
-    }
-  }
 
-  componentDidMount() {
-    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
-      .then(response => response.json())
-      .then(movies => this.setState({ movies }))
-      // .then(movies => console.log(this.state))
-      .catch(error => console.log('Movie not found'))
-  }
 
-  render() {
-    console.log('lives in card section', this.state.movies)
-    const allMovies = this.state.movies.map(movie => {
+function CardSection(props) {
+  let movies;
+  console.log('length', props.allMovies.movies)
+  if (props.allMovies.length > 0) {
+    console.log('test', props)
+    movies = props.allMovies.movies.map(movie => {
       return (
-      <MovieCard movie={movie} />
-    )
-  })
-    return (
-      <section>
```

Comment:
I think that we can refactor our map to be directly placed within the return statement when we render the card section. Something to think of once we figure out our current issue.